docs/preview/index.md

```diff
@@ -24,6 +24,7 @@ PM > Install-Package Arcus.Security.Providers.AzureKeyVault
     - [Azure Key Vault](features/secret-store/provider/key-vault)
     - [Configuration](features/secret-store/provider/configuration)
     - [Environment variables](features/secret-store/provider/environment-variables)
+    - [Docker secrets](features/secret-store/provider/docker-secrets)
```

Comment:
Nit - Can you move it before Environment variables please?

Reply:
Ah, didn't know they had to be in alphabetical order.  Changed.